aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/AWSPinpointAnalyticsPluginConfiguration.java

```diff
@@ -15,23 +15,27 @@
 
 package com.amplifyframework.analytics.pinpoint;
 
+import androidx.annotation.NonNull;
+
+import com.amplifyframework.AmplifyException;
+import com.amplifyframework.analytics.AnalyticsException;
+import com.amplifyframework.core.configuration.AmplifyOutputsData;
+
 /**
  * Configuration options for Amplify Analytics Pinpoint plugin.
  */
 final class AWSPinpointAnalyticsPluginConfiguration {
 
-    private static final long DEFAULT_AUTO_FLUSH_INTERVAL = 30000L;
+    static final long DEFAULT_AUTO_FLUSH_INTERVAL = 30000L;
 
     // Pinpoint plugin configuration options
     private final String appId;
-    private final boolean trackAppLifecycleEvents;
     private final String region;
     private final long autoFlushEventsInterval;
 
     private AWSPinpointAnalyticsPluginConfiguration(Builder builder) {
         this.appId = builder.appId;
         this.region = builder.region;
-        this.trackAppLifecycleEvents = builder.trackAppLifecycleEvents;
```

Comment:
A bit confused here. Is this work removing this capability for our gen1 customers?

Reply:
No this is actually dead code - we never actually used this flag for anything, as lifecycle tracking is always enabled and cannot be disabled.